lib/v2/comicat/search.js

```diff
@@ -0,0 +1,43 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+const baseUrl = 'https://comicat.org';
+
+module.exports = async (ctx) => {
+    const keyword = ctx.params.keyword;
+    const { data: response } = await got(`${baseUrl}/search.php?keyword=${encodeURIComponent(keyword)}}`);
+    const $ = cheerio.load(response);
+    const list = $('#listTable tbody > tr')
+        .toArray()
+        .map((item) => ({
+                title: $(item).find('td:nth-child(3)').text().trim(),
+                link: `${baseUrl}/${$(item).find('td:nth-child(3) a').attr('href')}`,
+                pubDate: parseDate($(item).find('td:nth-child(1)').text().trim()),
```

Comment:
Parse the pubDate at item level instead. The first two pubDate in automated test are invalid.

Reply:
When I move pubDate into item level, the invalid problem seems gone.